docs/topics/README.md

```diff
@@ -12,8 +12,9 @@ Introductions to all the key parts of AKS-Engine you’ll need to know.
 - [For Kubernetes Developers](kubernetes-developers.md)
 - [Kubernetes Walkthrough](kubernetes-walkthrough.md)
 - [Monitoring Kubernetes Clusters](monitoring.md)
-- [Scaling Kubernetes Clusters](scaling.md)
+- [Scaling Kubernetes Clusters](scale.md)
 - [Service Principals](service-principals.md)
+- [Upgrading Kubernetes Clusters](upgrade.md)

```

Comment:
Let's move this to the top of the list.

Reply:
I think the list is meant to be alphabetical

Reply:
Indeed. And the result of that is that we're sort of obscuring upgrade/scale docs, which (IMO) should be emphasized more prominently. Let's give them the clickbait level up they deserve.

Reply:
I don't agree here. One could argue `Features` or `Cluster Definition` are as if not more important then upgrade and scale. I think @bacongobbler's new organization of topics actually makes it easier for users to find what they are looking for. Keeping the list organized ensures it doesn't rapidly become a long list of unrelated items so I think alphabetical order makes sense here.

Reply:
I also "advertised" the upgrade doc in the troubleshooting doc, we could do the same for scale